ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -566,14 +548,18 @@ void pdo_sqlite_create_aggregate_internal(INTERNAL_FUNCTION_PARAMETERS)
 	pdo_dbh_t *dbh;
 	pdo_sqlite_db_handle *H;
 	int ret;
+	bool is_throw = false;
+
+	step_fcc.function_handler = NULL;
+	fini_fcc.function_handler = NULL;
 
 	ZEND_PARSE_PARAMETERS_START(3, 4)
 		Z_PARAM_STRING(func_name, func_name_len)
-		Z_PARAM_FUNC(step_fci, step_fcc)
-		Z_PARAM_FUNC(fini_fci, fini_fcc)
+		Z_PARAM_FUNC_NO_TRAMPOLINE_FREE(step_fci, step_fcc)
+		Z_PARAM_FUNC_NO_TRAMPOLINE_FREE(fini_fci, fini_fcc)
```

Comment:
Ditto, also are there trampoline tests?

Reply:
This one should probably be okay. But I will check if there is a test to check for leaks. (If it doesn't exist, I create it)

Reply:
By the way, the trampoline test itself does exist. However, the behavior when passing invalid arguments is not checked.
I feel like I can check it with `arginfo_zpp_mismatch.phpt`, but I'll take a closer look.

Reply:
Added https://github.com/php/php-src/pull/14059/commits/0ad72727fd65c0b1bf5e9de7e7fbfec4e274c138